configuration/tcpip/README.md

```diff
@@ -0,0 +1,23 @@
+# TCP/IP networking
+
+The Raspberry Pi uses dhcpcd to configure TCP/IP across all of its network interfaces. The dhcpcd daemon was written by Roy Marples and is intended to be an all-in-one ZeroConf client for UNIX-like systems. This includes assigning each interface an IP address, setting netmasks and configuring DNS resolution via the Name Service Switch (NSS) facility. 
+
+By default, Raspbian attempts to automatically configure all network interfaces by DHCP, falling back to automatic private addresses in the range 169.254.0.0/16 if DHCP fails. This is consistent with the behaviour of other Linux variants, and Microsoft Windows.
+
+## Static IP address
+
+If you wish to disable automatic configuration for an interface and instead configure it statically, add the details to `/etc/dhcpcd.conf`, for example:
+
+```interface eth0
```

Comment:
There is a formatting bug here, hiding the "interface eth0" line.

Reply:
Good spot, there should be a linefeed after the three backticks.